mcs/build/tests.make

```diff
@@ -308,9 +308,9 @@ check: run-xunit-test-local
 xunit-test-local: $(xunit_test_lib)
 run-xunit-test-local: run-xunit-test-lib
 
-# cp -rf is a HACK for xunit runner to require xunit.execution.desktop.dll file in local folder on .net only
+# cp -rf is a HACK for xunit runner to require xunit.execution.dOTNET.dll file in local folder on .net only
 run-xunit-test-lib: xunit-test-local $(XTEST_REMOTE_EXECUTOR)
-	@cp -rf $(XTEST_HARNESS_PATH)/xunit.execution.desktop.dll xunit.execution.desktop.dll
+	@cp -rf $(XTEST_HARNESS_PATH)/xunit.execution.dotnet.dll xunit.execution.dotnet.dll
```

Comment:
@marek-safar a few lines below we still do `@rm -f xunit.execution.desktop.dll`.

can we delete xunit.execution.desktop.dll and xunit.runner.utility.net452.dll from the xunit-binaries submodule?

Reply:
Will fix the name mismatch but we still need ` xunit.runner.utility.net452.dll` because it's the console runner dependency